client-app/shared/checkout/composables/useCheckout.ts

```diff
@@ -459,6 +466,10 @@ export function _useCheckout() {
     return placedOrder.value;
   }
 
+  function clearState() {
```

Comment:
The point of creating this function is not clear - can we just call `clearGlobalCheckoutState` on the line **451**?

Reply:
The useGlobalCheckout is a separate entity and has there own state.
The useCheckout also is a separate entity with their own state.
The idea is to have the same signature in every entity.

Reply:
Ok, I got it, thanks for explaining :)